src/concrete/ml/quantization/quantized_ops.py

```diff
@@ -174,38 +168,49 @@ def q_impl(
         alpha = self.attrs.get("alpha", 1)
         beta = self.attrs.get("beta", 1)
 
+        # If self.constant_inputs is empty this is an encrypted gemm
+        is_encrypted_gemm = isinstance(self.constant_inputs, dict) and not self.constant_inputs
+
         # If alpha != 1 or beta not in [0, 1], this function must be modified
         assert_true(alpha == 1)
-        assert_true(beta in [0, 1])
+        assert_true(beta in {0, 1})
```

Comment:
why this change ?

Reply:
Performance wise, checking if a value is in a dict is much better than checking if it's in a list. Here it's only 2 values so I guess it's doesn't change much but still good practice to have a dict when doing such comparison I think. That's a change made by @fd0r btw

Reply:
you're right indeed, but yes for 2 values it's not important